chemprop/cli/common.py

```diff
@@ -93,11 +93,13 @@ def add_common_args(parser: ArgumentParser) -> ArgumentParser:
     )
     featurization_args.add_argument(
         "--atom-features-path",
-        help="Path to the extra atom features. Used as atom features to featurize a given molecule.",
+        nargs="+",
+        help="Path to the extra atom features. Used as atom features to featurize a given molecule. Use as --atom-features-path 0 /path/to/atom/features/for/mol/0.npz 2 /path/to/atom/features/for/mol/2.npz",
```

Comment:
though this should be in a different PR, I'll comment here so you see it:
```suggestion
        nargs=2,
        action="append",
        help="A two-tuple of component index and path to additional atom features to supply after message passing. (instead of features_generator). Ex: `--features-path 0 /path/to/features_0.npz` indicates that the features at the given path should be supplied to the 0-th component. To supply additional features for multiple components, repeat this argument on the command line for each component's respective values, e.g., `--features-path [...] --features-path [...]`.",
```